showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/logging/ITLogging.java

```diff
@@ -0,0 +1,207 @@
+  @BeforeAll
+  static void createClients() throws Exception {
+    // Create gRPC Echo Client
+    grpcClient = TestClientInitializer.createGrpcEchoClient();
+    // Create Http JSON Echo Client
+    httpjsonClient = TestClientInitializer.createHttpJsonEchoClient();
+  }
+
+  @AfterAll
+  static void destroyClients() throws InterruptedException {
+    grpcClient.close();
+    httpjsonClient.close();
+
+    grpcClient.awaitTermination(TestClientInitializer.AWAIT_TERMINATION_SECONDS, TimeUnit.SECONDS);
+    httpjsonClient.awaitTermination(
+        TestClientInitializer.AWAIT_TERMINATION_SECONDS, TimeUnit.SECONDS);
+  }
+
+  @Test
+  void testGrpc_receiveContent_logDebug() {
+    TestAppender testAppender = setupTestLogger(GrpcLoggingInterceptor.class);
+    assertThat(echoGrpc(ECHO_STRING)).isEqualTo(ECHO_STRING);
+
+    assertThat(true).isTrue();
+
+    assertThat(testAppender.events.size()).isEqualTo(2);
+    // logging event for request
+    ILoggingEvent loggingEvent1 = testAppender.events.get(0);
+    assertThat(loggingEvent1.getMessage()).isEqualTo("Sending gRPC request");
+    assertThat(loggingEvent1.getLevel()).isEqualTo(Level.DEBUG);
+    List<KeyValuePair> keyValuePairs = loggingEvent1.getKeyValuePairs();
+    assertThat(keyValuePairs.size()).isEqualTo(4);
+    assertThat(keyValuePairs).containsAtLeast(SERVICE_NAME_KEY_VALUE_PAIR, RPC_NAME_KEY_VALUE_PAIR);
+
+    for (KeyValuePair kvp : keyValuePairs) {
+      if (kvp.key.equals("request.payload")) {
+        Map payload = (Map) kvp.value;
+        assertThat(payload.get("content")).isEqualTo(ECHO_STRING);
+        assertThat(payload.get("requestId")).isNotNull();
+        assertThat(payload.get("otherRequestId")).isNotNull();
+      }
+      if (kvp.key.equals("request.headers")) {
+        Map headers = (Map) kvp.value;
+        assertThat(headers).containsKey("x-goog-api-version");
+        assertThat(headers).containsKey("x-goog-api-client");
+      }
+    }
+
+    // logging event for response
+    ILoggingEvent loggingEvent2 = testAppender.events.get(1);
+    assertThat(loggingEvent2.getMessage()).isEqualTo("Received Grpc response");
+
+    assertThat(loggingEvent2.getLevel()).isEqualTo(Level.DEBUG);
+    List<KeyValuePair> keyValuePairs2 = loggingEvent2.getKeyValuePairs();
+    assertThat(keyValuePairs2.size()).isEqualTo(5);
+    assertThat(keyValuePairs2)
+        .containsAtLeast(
+            RESPONSE_STATUS_KEY_VALUE_PAIR,
+            RESPONSE_HEADERS_KEY_VALUE_PAIR,
+            SERVICE_NAME_KEY_VALUE_PAIR,
+            RPC_NAME_KEY_VALUE_PAIR);
+    for (KeyValuePair kvp : keyValuePairs2) {
+      if (kvp.key.equals("response.payload")) {
+        Map payload = (Map) kvp.value;
+        assertThat(payload.get("content")).isEqualTo(ECHO_STRING);
+        assertThat(payload.get("requestId")).isNotNull();
+        assertThat(payload.get("otherRequestId")).isNotNull();
+      }
+    }
+    testAppender.stop();
+  }
+
+  @Test
+  void testHttpJson_receiveContent_logDebug() {
```

Comment:
could there also be `logInfo` equivalent test case well?

Reply:
added in [7c1ea76](https://github.com/googleapis/sdk-platform-java/pull/3403/commits/7c1ea762649703ceff780bc2b72b564d0788169c)